projects/packages/my-jetpack/_inc/components/product-card/style.module.scss

```diff
@@ -1,112 +1,119 @@
-  --product-card-actions-font-size: 12px;
-  --product-card-margin-base: 4px;
-  --product-card-actions-size: 36px;
-  --status-size: 8px;
-  --status-active: #008710;
-  --status-inactive: #646970;
-  --status-error: #B32D2E;
-  --status-plugin_absent: #C3C4C7;
+	// all css variables used down in product-card
+	--product-card-shadow: rgba(0, 0, 0, 0.08);
+	--product-card-shadow-size: 40px;
+	--product-card-name-font-size: 24px;
+	--product-card-description-font-size: 14px;
+	--product-card-actions-font-size: 12px;
+	--product-card-margin-base: 4px;
+	--product-card-actions-size: 36px;
+	--jp-black: #000;
+	--status-size: 8px;
+	--status-active: #008710;
+	--status-inactive: #646970;
+	--status-error: #B32D2E;
+	--status-plugin_absent: #C3C4C7;
+	--jp-underline-thickness: 2px;
 
-  padding: 24px;
-  background: var( --jp-white );
-  border-radius: var( --jp-border-radius );
-  box-shadow: 0 0 var(--product-card-shadow-size) var(--product-card-shadow);
-  height: 100%;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
+	padding: 24px;
+	background: var( --jp-white );
+	border-radius: var( --jp-border-radius );
+	box-shadow: 0 0 var(--product-card-shadow-size) var(--product-card-shadow);
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
 
-  &.is-purchase-required,
-  &.plugin_absent {
-    background: none;
-    box-shadow: none;
-    box-shadow: 0 0 0 1px var( --status-plugin_absent ) inset;
-  }
+	&.is-purchase-required,
+	&.plugin_absent {
+		background: none;
+		box-shadow: none;
+		box-shadow: 0 0 0 1px var( --status-plugin_absent ) inset;
+	}
 }
 
 .name {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  width: 100%;
-  font-size: var(--product-card-name-font-size);
-  font-weight: 700;
-  margin-bottom: var(--product-card-margin-base);
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	width: 100%;
+	font-size: var(--product-card-name-font-size);
+	font-weight: 700;
+	margin-bottom: var(--product-card-margin-base);
 }
 
 .description {
-  font-size: var(--product-card-description-font-size);
-  margin:0;
+	font-size: var(--product-card-description-font-size);
+	margin:0;
 }
 
 .actions {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  width: 100%;
-  min-height: var(--product-card-actions-size);
-  margin-top: calc(var(--product-card-margin-base) * 4); // 16px
-  font-size: var(--product-card-actions-font-size);
-  font-weight: 600;
-  flex-wrap: wrap;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	width: 100%;
+	min-height: var(--product-card-actions-size);
+	margin-top: calc(var(--product-card-margin-base) * 4); // 16px
+	font-size: var(--product-card-actions-font-size);
+	font-weight: 600;
+	flex-wrap: wrap;
+
+	.action-link-button:hover {
+		color: var(--jp-black);
+		text-decoration-thickness: var(--jp-underline-thickness);
+	}
```

Comment:
You don't need to put this one inside the `.actions` body, it could live outside, since `CSS Modules` will hash it.

Reply:
🤷

Reply:
actually, you're right. updated 👍